src/components/Person/Person.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name, age, sex, isMarried, partnerName: pName,
+  } = person;
+
+  return (
+    // eslint-disable-next-line react/jsx-filename-extension
+    <section className="Person">
```

Comment:
Use small letters in class naming
```suggestion
    <section className="person">
```